Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -882,7 +884,7 @@ AccessibilityObject* AXObjectCache::getOrCreate(Node& node, IsPartOfRelation isP
         auto* element = dynamicDowncast<Element>(node);
         bool hasDisplayContents = element && element->hasDisplayContents();
         bool isPopover = element && element->hasAttributeWithoutSynchronization(popoverAttr);
-        if (!inCanvasSubtree && !insideMeterElement && !hasDisplayContents && !isPopover && !isNodeAriaVisible(node))
+        if (!inCanvasSubtree && !insideMeterElement && !hasDisplayContents && !isPopover && !isNodeFocused(node))
```

Comment:
Do we need to use `isNodeFocused()` here? We already do the work to cast to `Element`. What about:

```
&& (!element || !element->focused())`
```

Reply:
Yeah this should work, we might be able to get rid of `isNodeFocused` altogether.